src/Controls/samples/Controls.Sample.UITests/Issues/Issue17347.cs

```diff
@@ -4,14 +4,15 @@
 namespace Maui.Controls.Sample.Issues
 {
 	[Issue(IssueTracker.Github, 17347, "Setting a new TitleView on an already created page crashes iOS", PlatformAffected.iOS)]
-	public class Issue17347 : TestContentPage
+	public class Issue17347 : NavigationPage
```

Comment:
Could we create a TestContentNavigationPage that doe the ctor stuff automatically and adds a Content property that does the same things as the casts. Then we don't need to duplicate all the logic everywhere.

Reply:
I split out the places I had this to separate pages so it's less weird looking. LMK if that seems alright